lib/mpl_toolkits/axisartist/angle_helper.py

```diff
@@ -54,19 +46,11 @@ def select_step_hour(dv):
     second_limits_ = np.array(minsec_limits_) / 3600
     second_factors = [3600.] * len(second_limits_)
 
-    hour_limits = np.concatenate([second_limits_,
-                                  minute_limits_,
-                                  hour_limits_])
-
-    hour_steps = np.concatenate([minsec_steps_,
-                                 minsec_steps_,
-                                 hour_steps_])
-
-    hour_factors = np.concatenate([second_factors,
-                                   minute_factors,
-                                   hour_factors])
+    hour_limits = [*second_limits_, *minute_limits_, *hour_limits_]
+    hour_steps = [*minsec_steps_, *minsec_steps_, *hour_steps_]
+    hour_factors = [*second_factors, *minute_factors, *hour_factors]
 
-    n = hour_limits.searchsorted(dv)
+    n = np.searchsorted(hour_limits, dv)
     step = hour_steps[n]
```

Comment:
How does this work? `n` is an array still, but `hour_steps` is a list now, and I can't index a list with an array.

Reply:
dv is a scalar (this is called by select_step, where it is immediately preceded by `if dv > 1 / threshold_factor` -- semantically, it is the distance between the axes bounds) so n is a scalar as well and the indexing is fine.

I guess *technically* it is an API break in that select_step_degree *could* have been called directly by someone passing dv as an array, but I don't think this is a real issue (and is clearly not how select_step_degree was designed to do).

Reply:
s/preceded/succeeded/, but I see what you mean.